payu/models/cesm_cmeps.py

```diff
@@ -190,6 +185,76 @@ def setup(self):
                 # TODO: copied this from other models. Surely we want to exit here or something
                 print('payu: error: Unable to find mod_def.ww3 file in input directory')
 
+    def _setup_checks(self):
+        # check pelayout fits within requested cpucount
+        cpucount = int(
+            self.expt.config.get('ncpus', multiprocessing.cpu_count())
```

Comment:
Multiprocessing is not designed to be used on multiple nodes

https://docs.nersc.gov/development/languages/python/parallel-python/#multiprocessing

Consequently I don't think `multiprocessing.cpu_count()` will return useful information and you should just insist that the user set this value and error out otherwise.

Reply:
Ok done